src/components/cc-zone-picker/cc-zone-picker.stories.js

```diff
@@ -0,0 +1,255 @@
+    flagUrl: getFlagUrl('AU'),
+    countryCode: 'AU',
+    country: 'Australia',
+  },
+  {
+    code: 'rbx',
+    name: 'Roubaix',
+    images: [{ url: ovhIcon, alt: 'infra' }],
+    flagUrl: getFlagUrl('FR'),
+    countryCode: 'FR',
+    country: 'France',
+  },
+  {
+    code: 'wsw',
+    name: 'Warsaw',
+    images: [{ url: ovhIcon, alt: 'infra' }],
+    flagUrl: getFlagUrl('PL'),
+    countryCode: 'PL',
+    country: 'Poland',
+  },
+  {
+    code: 'rbxhds',
+    name: 'Roubaix',
+    images: [{ url: ovhIcon, alt: 'infra' }],
+    flagUrl: getFlagUrl('FR'),
+    countryCode: 'FR',
+    country: 'France',
+  },
+  {
+    code: 'fr-north-hds',
+    name: 'North',
+    images: [{ url: ovhIcon, alt: 'infra' }],
+    flagUrl: getFlagUrl('FR'),
+    countryCode: 'FR',
+    country: 'France',
+  },
+];
+
+/** @type {ZoneItem[]} */
+const PRIVATE_ZONES = [
+  {
+    code: 'foo-foobars',
+    name: 'Private MySQL Cluster',
+    images: [],
+    flagUrl: getFlagUrl('FR'),
+    country: 'France',
+    countryCode: 'FR',
+  },
+  {
+    code: 'Foo',
+    name: 'City Member Lab',
+    images: [],
+    flagUrl: getFlagUrl('FR'),
+    country: 'France',
+    countryCode: 'FR',
+  },
+  {
+    code: 'foo5',
+    name: 'testing environment',
+    images: [],
+    flagUrl: getFlagUrl('FR'),
+    country: 'France',
+    countryCode: 'FR',
+  },
+  {
+    code: 'foobarz',
+    name: 'Foobarz',
+    images: [],
+    flagUrl: getFlagUrl('FR'),
+    country: 'France',
+    countryCode: 'FR',
+  },
+  {
+    code: 'foozbar',
+    name: 'Chips Dale Sound City',
+    images: [],
+    flagUrl: getFlagUrl('FR'),
+    country: 'France',
+    countryCode: 'FR',
+  },
+  {
+    code: 'foobazbabarzone',
+    name: 'Sleep Edge Abroad Bird Random',
+    images: [],
+    flagUrl: getFlagUrl('FR'),
+    country: 'France',
+    countryCode: 'FR',
+  },
+  {
+    code: 'champion',
+    name: 'Private MongoDB Cluster',
+    images: [],
+    flagUrl: getFlagUrl('FR'),
+    country: 'France',
+    countryCode: 'FR',
+  },
+  {
+    code: 'ichipsndales-postgresql-internal',
+    name: 'Private PostgreSQL Cluster',
+    images: [],
+    flagUrl: getFlagUrl('FR'),
+    country: 'France',
+    countryCode: 'FR',
+  },
+  {
+    code: 'mainstream',
+    name: 'Sleep Edge Abroad Bird Matrix',
+    images: [],
+    flagUrl: getFlagUrl('FR'),
+    country: 'France',
+    countryCode: 'FR',
+  },
+];
+
+export const defaultStory = makeStory(conf, {
+  items: [
+    {
+      zonesSection: [
+        {
+          title: '',
+          zones: PUBLIC_ZONES,
+        },
+      ],
+      value: 'par',
+    },
+  ],
+});
+
+export const publicAndPrivateZones = makeStory(conf, {
+  items: [
+    {
+      zonesSection: [
+        {
+          title: 'Private zones',
+          zones: PRIVATE_ZONES,
+        },
+        {
+          title: 'Public zones',
+          zones: PUBLIC_ZONES,
+        },
+      ],
+    },
+  ],
+});
+
+export const multipleSection = makeStory(conf, {
+  items: [
```

Comment:
**suggestion:** I think you would benefit from typing this as well, for instance by adding this
```suggestion
  /** @type {Array<Partial<CcZonePicker>>} */  
  items: [
```

We need to type `makeStory` eventually so we don't have to do this but in the meantime, this can make refactoring easier + it adds some type checking.
For instance, if we decide `zoneSection` should be `zoneSections`, renaming the prop within the component should also rename the prop here :tada: